apps/site/src/components/dom/SceneLayout.tsx

```diff
@@ -37,4 +37,4 @@ const Layout = ({ children }: { children: ReactNode }) => {
 	);
 };
 
-export { Layout };
+export { SceneLayout };
```

Comment:
Nitpick [partially out of scope]: Since this is the only thing that's exported from this file, we should make `SceneLayout` a default export.

Reply:
Kinda feeling (in general) that having named exports helps preserve references, but that's also out of scope.